example/fenl/KokkosKernels_ETIMacros.hpp

```diff
@@ -1,17 +1,17 @@
 #ifdef MAKE_BUILD
-#ifdef KOKKOS_HAVE_CUDA
+#ifdef KOKKOS_ENABLE_CUDA
   #define KOKKOSKERNELS_ETI_MANGLING_TYPEDEFS()  \
                         typedef Kokkos::Device<Kokkos::Cuda, Kokkos::Cuda::memory_space> Kokkos_Device0Kokkos_Cuda_Kokkos_CudaSpace0; \
         typedef Kokkos::complex<double> Kokkos_complex0double0; \
         typedef long long longlong;
 #else
-  #ifdef KOKKOS_HAVE_OPENMP
+  #ifdef KOKKOS_ENABLE_OPENMP
     #define KOKKOSKERNELS_ETI_MANGLING_TYPEDEFS()  \
                         typedef Kokkos::Device<Kokkos::OpenMP, Kokkos::OpenMP::memory_space> Kokkos_Device0Kokkos_OpenMP_Kokkos_HostSpace0; \
         typedef Kokkos::complex<double> Kokkos_complex0double0; \
         typedef long long longlong;
   #else
-    #ifdef KOKKOS_HAVE_PTHREAD
+    #ifdef KOKKOS_ENABLE_PTHREAD
```

Comment:
I think this needs to be KOKKOS_ENABLE_THREADS?

Reply:
Yeah I checked this needs to KOKKOS_ENABLE_THREADS

Reply:
Fixed all occurrences and pushed.